main.js

```diff
@@ -159,3 +159,40 @@ form.addEventListener('submit', (e) => {
   document.getElementById('err-display').innerHTML = text;
   document.getElementById('err-display').style.border = '1px solid rgba(255, 0, 0, 0.9)';
 });
+
+// Local storage
+
+// Select all form inputs
+const formInputs = document.querySelectorAll('.input-cls');
+const formInputsArr = Array.from(formInputs);
+
+// Select form sbmit button
+const submitBtn = document.querySelector('.submit-btn');
+
+// select localstorage to check if there is already existing data
+const localData = localStorage;
+
+// Object to store our collected data
+const ContactForm = {
+  fullName: '',
+  email: '',
+  message: '',
+};
+
+// Check if there is data and if its there, fill the inputs with it
+if (localData.fullName || localData.email || localData.message) {
```

Comment:
- [ ] As suggested by the project requirements. I kindly suggest you make use of a single object stored to a single local storage registry.

- you can make use of `windows.onload` listener to save data on page reload
## Example
```
const formInputsArr = Array.from(formInputs);
window.onload = () => {
  const  ContactForm = JSON.parse(localStorage.getItem('form'));

  if (ContactForm) {
    formInputsArr[0].value =ContactForm.fullName;
    formInputsArr[1].value =  ContactForm.email;
    formInputsArr[2].value = ContactForm.message;
  }
};
```